### ISSUE TITLE
DID NOT get the comparable results as that in your PAPER,  WHY?

### ISSUE THREAD

**Issue by ZhangSanFengByGit** [foolwood/DaSiamRPN#33]
DID NOT get the comparable results as that in your PAPER,  WHY?
I load your DaSiam.model in the pytorch and run the whole vot datasets(Both vot2016 and vot2017), and Sadly I did not even a comparable result by the Normal international criterion in Video Tracking, WHY?

**Comment by ZhangSanFengByGit** [comment]
I mean when load  ' *SiamRPNBIG.model'    pretrain model

**Comment by xujiao3628** [comment]
> I mean when load ' *SiamRPNBIG.model' pretrain model

The pretrained model with which author provided is different from the model in paper.

**Comment by ZhangSanFengByGit** [comment]
> > I mean when load ' *SiamRPNBIG.model' pretrain model
> 
> The pretrained model with which author provided is different from the model in paper.

Then do you know where is the model in the paper?
OR Why he provided a model with a low precision result, please~
Thx

**Comment by xujiao3628** [comment]
> Why he provided a model with a low precision result, please~
> Thx

sorry,I don't know.

**Comment by ucasqcz** [comment]
> > > I mean when load ' *SiamRPNBIG.model' pretrain model
> > 
> > 
> > The pretrained model with which author provided is different from the model in paper.
> 
> Then do you know where is the model in the paper?
> OR Why he provided a model with a low precision result, please~
> Thx

do you get the otb2013 result files?

**Comment by ZhangSanFengByGit** [comment]
> > > > I mean when load ' *SiamRPNBIG.model' pretrain model
> > > 
> > > 
> > > The pretrained model with which author provided is different from the model in paper.
> > 
> > 
> > Then do you know where is the model in the paper?
> > OR Why he provided a model with a low precision result, please~
> > Thx
> 
> do you get the otb2013 result files?

what? I just use the model to test vot2016 and 2017 and the results are pretty bad

**Comment by ucasqcz** [comment]
> > > > > I mean when load ' *SiamRPNBIG.model' pretrain model
> > > > 
> > > > 
> > > > The pretrained model with which author provided is different from the model in paper.
> > > 
> > > 
> > > Then do you know where is the model in the paper?
> > > OR Why he provided a model with a low precision result, please~
> > > Thx
> > 
> > 
> > do you get the otb2013 result files?
> 
> what? I just use the model to test vot2016 and 2017 and the results are pretty bad
google driver provides the result of OTB100 in txt file, i have verified that the txt file result is correct, but the model released by author only obtains auc 0.558 on OTB100

**Comment by ZhangSanFengByGit** [comment]
> > > > > > I mean when load ' *SiamRPNBIG.model' pretrain model
> > > > > 
> > > > > 
> > > > > The pretrained model with which author provided is different from the model in paper.
> > > > 
> > > > 
> > > > Then do you know where is the model in the paper?
> > > > OR Why he provided a model with a low precision result, please~
> > > > Thx
> > > 
> > > 
> > > do you get the otb2013 result files?
> > 
> > 
> > what? I just use the model to test vot2016 and 2017 and the results are pretty bad
> > google driver provides the result of OTB100 in txt file, i have verified that the txt file result is correct, but the model released by author only obtains auc 0.558 on OTB100

you mean that you have verified that the Precision and SuccessRate results on OTB100 is as the same as that in the paper?

**Comment by ZhangSanFengByGit** [comment]
@foolwood

**Comment by foolwood** [comment]
@ZhangSanFengByGit @xujiao3628 @ucasqcz 
Sorry for the late reply.

This [repository](https://github.com/foolwood/DaSiamRPN) is used to reproduce the results of VOT2018 challenge (EAO:0.3827).
The [SiamRPNBIG.model](https://drive.google.com/file/d/1-vNVZxfbIplXHrqMHiJJYWXYWsOIvGsf/view?usp=sharing) focuses on the bounding box regression for achieving high accuracy on VOT2018. This model is different from the model used in ECCV version.

Our `SiamRPNOTB.model` is released on [Google Drive](https://drive.google.com/open?id=1BtIkp5pB6aqePQGlMb2_Z7bfPy6XEj6H).
The AUC on OTB2015 is 0.655 and the speed is 200fps with GTX 1080ti, which is the **fastest** Siamese Tracker.

Thanks for your question.
